blockchain/state/statedb.go

```diff
@@ -1040,10 +1040,6 @@ func (s *StateDB) Commit(deleteEmptyObjects bool) (root common.Hash, err error)
 			if pa.GetStorageRoot() != emptyState {
 				s.db.TrieDB().Reference(pa.GetStorageRoot(), parent)
 			}
-			code := common.BytesToHash(pa.GetCodeHash())
```

Comment:
Is this removal is necessary?

Reply:
Since we insert the code into database directly (in `WriteCode` instead of `InsertBlob`), these code fragments are useless in terms of caching and pruning.

Also, there is a discussion about this in the original PR as well. https://github.com/ethereum/go-ethereum/pull/21080/files#r465045551.